src/gs-details-page.ui

```diff
@@ -153,15 +153,18 @@
                             <property name="margin_bottom">14</property>
                             <property name="spacing">9</property>
                             <child>
-                              <object class="GtkLinkButton" id="button_details_add_shortcut">
+                              <object class="GtkButton" id="button_details_add_shortcut">
                                 <property name="visible">False</property>
                                 <property name="can_focus">True</property>
                                 <property name="use_underline">True</property>
                                 <property name="label" translatable="yes" comments="Translators: A label for a button to add a shortcut to the selected application.">_Add to Desktop</property>
+                                <style>
+                                  <class name="suggested-action"/>
```

Comment:
Does this need to change dynamically as different buttons are made visible/hidden?

Reply:
Not sure why being dynamic here? I have taken this change from `eos3.6` branch (i.e. this is a fallout from the 3.32 rebase.) 

Both `_add_shortcut` and `_remove_shortcut` are separate buttons and visibility is toggled according to the given situation in the icon-grid. Not sure why we need to make the style class being dynamically applied

Reply:
As in, buttons on the details page are often shown/hidden based on the state of the app. We don’t ever want the situation where *two* buttons are visible, both with class `suggested-action`.

I don’t know if that’s the case after applying this PR, but please check.

Reply:
No, that's not the case. I can see it in action. It is making the behavior consistent with what we have/had in `3.6` OS's gnome-software. 

(I have a 3.6 release OS image VM which I use to compare and contrast these UI fixes).

Reply:
I think Philip's point is that both this button, and "Launch", have the `suggested-action` style class. Won't this mean that, when the app is installed but not on the desktop, both "Launch" and "Add to Desktop" will be blue? Could you take screenshots of the 3 states (not installed, installed but not on desktop, installed and on desktop)?

Reply:
**Not installed**
![Screenshot from 2019-08-01 20-08-39](https://user-images.githubusercontent.com/8775041/62302645-596f3600-b498-11e9-9626-b3d37b682f58.png)


**Installed and not on desktop** 
![Screenshot from 2019-08-01 20-08-15](https://user-images.githubusercontent.com/8775041/62302670-61c77100-b498-11e9-8620-b39819377954.png)


**Installed and  on desktop**
![Screenshot from 2019-08-01 20-08-22](https://user-images.githubusercontent.com/8775041/62302684-668c2500-b498-11e9-94c1-d1fadab2245a.png)

Reply:
https://developer.gnome.org/hig/stable/buttons.html.en

> Each view should only include a single suggested or destructive button.

De-emphasising Launch was a deliberate downstream change, see discussion in https://phabricator.endlessm.com/T14949 and designs linked there. I think we should just drop the `suggested-action` on the launch button.

Reply:
P

> https://developer.gnome.org/hig/stable/buttons.html.en
> 
> > Each view should only include a single suggested or destructive button.
> 
> De-emphasising Launch was a deliberate downstream change, see discussion in https://phabricator.endlessm.com/T14949 and designs linked there. I think we should just drop the `suggested-action` on the launch button.

Pushed a `fixup!` which is linking where this particular changed should have been introduced downstream. Also, made a note in the commit message of the original commit_id on `eos3.6` branch.